CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 ## 4.23.0 (Unreleased)
+NOTES:
+* clouddeploy: updated example documentation ([#4344](https://github.com/hashicorp/terraform-provider-google-beta/pull/4344))
```

Comment:
do we need this? i have no preference, but we typically don't add doc changes to the changelog, right?

Reply:
removed